### PR TITLE
Microsoft active directory takeover and engage

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add Microsoft active directory takeover and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/microsoft-active-directory](http://0.0.0.0:8001/engage/microsoft-active-directory)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to u.c and refresh the page until you see the Microsoft Active Directory takeover
- Click on "Download the whitepaper" to access the landing page
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/web-squad/issues/1198)
- Make sure it complies with the [brief](https://docs.google.com/document/d/1mj6nl2MKIn8gSGnwkX8e4YRT6wXdK6ITvOqr4XgHWtU/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1195

### CODE REVIEW

**Comment by solazio** [comment]
@WendyS1000 please review this PR
https://www-ubuntu-com-canonical-web-and-design-pr-5252.run.demo.haus/
https://www-ubuntu-com-canonical-web-and-design-pr-5252.run.demo.haus/engage/microsoft-active-directory

File: templates/takeovers/_active-directory_takeover.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <h1>
+        Integrate Ubuntu Desktop with Active Directory
+      </h1>
+      <h4 class="p-heading--four">
+        A step by step guide to joining your desktops to Active Directory
+        domains.
+      </h4>
+      <a href="/engage/microsoft-active-directory?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_IOT_Desktop_Whitepaper_ActiveDirectory" class="p-button--positive u-hide--small"
+        ><span>Download the whitepaper</span></a
+      >
```
Comment (sowasred2012) [inline templates/takeovers/_active-directory_takeover.html]: the `<a>` tag is broken up unnecessarily here (compare to the `<a>` tag on line 24)

**Comment by anasereijo** [comment]
Takeover:
- [ ] can you move the word Active to the 2nd line? So it's next to Directory

Landing page:
- [ ] text to be `7-col` and image `5-col`
- [ ] centre image within col

**Comment by WendyS1000** [comment]
The takeover and landing page look great to me - thank you very much! They are both good to go live.

**Comment by solazio** [comment]
@anasereijo I can move the word Active on the 2nd line, but it will look odd on mobile devices... 
image

The text is already 7-col and the image is 5-col

**Comment by WendyS1000** [comment]
@anasereijo  and @solazio, I suggest rephrasing the text as follows 
Integrating Ubuntu Desktop with Active Directory 

Maybe it will work better across mobile/desktop. Or another option might be to use smaller headings on mobile - H3 perhaps. I realise that this will require dev work, but maybe something to consider:)

**Comment by anasereijo** [comment]
@solazio isn't it possible to break the text just on desktop and not on mobile? If not that's fine, don't break it.

sorry my mistake, on the landing page I meant to say to make the text 6-col and the image is 4-col

**Comment by anasereijo** [comment]
@WendyS1000 thanks for your suggestions!
@solazio can you try Wendy's new title?

thanks

**Comment by anasereijo** [comment]
QA -1: pdf isn't downloading

**Comment by WendyS1000** [comment]
@anasereijo, @solazio, please send me the takeover to preview with my new title suggestion. 
Also, there were changes to the PDF, so I've had to send a new one to Alexei to re-do the links. I will send the updated links to @SirSamTumless once I've got it. Have I used the right label here?

**Comment by solazio** [comment]
@WendyS1000 use this link https://www-ubuntu-com-canonical-web-and-design-pr-5252.run.demo.haus/
but note that this PR has already been merged

**Comment by SirSamTumless** [comment]
This wasn't supposed to go-live until the 13/Jun as per the due date and the Takeover schedule: https://docs.google.com/spreadsheets/d/1MaFd-ZHWpRVjIP9Sj_dOCIGOtl-GJYbrXewt5EG4m80/edit?ts=5cc05f70#gid=564832475. Please refer to this before changing the takeover rotation.

**Pull request merged**